tests/sentry/models/test_event.py

```diff
@@ -23,13 +23,27 @@ def test_email_subject(self):
             event_id='a' * 32, group=self.group, tags={'level': 'info'},
             message='Foo bar')
         event2 = self.create_event(
-            event_id='b' * 32, group=self.group, tags={'level': 'error'},
+            event_id='b' * 32, group=self.group, tags={'level': 'ERROR'},
             message='Foo bar')
         self.group.level = 30
 
-        assert event1.get_email_subject() == '[foo Bar] INFO: Foo bar'
+        assert event1.get_email_subject() == '[foo Bar] info: Foo bar'
```

Comment:
If anybody has a really strong opinion that this absolutely should be capitalized, I could special case this tag key with a post-processing function. (Though if that is the case, shouldn't we just normalize it as part of the ingestion pipeline so everything downstream is correct by default?)